templates/summary.html

```diff
@@ -1,11 +1,45 @@
-{% extends 'confirmation.html' %}
+{% extends 'layouts/_questionnaire.html' %}
+{% from "components/button/_macro.njk" import onsButton %}
 {% from "components/panel/_macro.njk" import onsPanel %}
 
 {% import 'macros/helpers.html' as helpers %}
 
 {% set summary_page_title = content.title %}
 {% set save_on_signout = true %}
 
+{% block submit_button %}
+  {% if content.summary and content.summary.summary_type == 'CalculatedSummary' %}
+    {{
+      onsButton({
+        "text": _("Yes, I confirm these are correct"),
+        "submitType": 'timer',
+        "classes": "btn--loader js-btn-submit",
+        "attributes": {
+          "data-qa": "btn-submit"
+        }
+      })
+    }}
+  {% else %}
+    {% if block and block.content %}
+      {% set content_block = block.content %}
+      {% include 'partials/content-block.html' %}
+    {% endif %}
+    <p class="u-mt-m">
+      {{
+        onsButton({
+          "text": content.submit_button if content.submit_button else _("Submit answers"),
+          "submitType": 'timer',
+          "classes": "btn--loader js-btn-submit",
+          "attributes": {
+            "data-qa": "btn-submit",
+            "data-loading-msg": _("Submitting") + "&hellip;"
+          }
+        })
+      }}
+    </p>
+  {% endif %}
+{% endblock %}
+
```

Comment:
I think the summary should be separated from the calculated summary template. We should consider the commonality between the hub/confirmation/submit when doing this.

Reply:
I think a small discussion is worthwhile before I dive into refactoring this.

Reply:
https://github.com/ONSdigital/eq-questionnaire-runner/pull/606#discussion_r635458345